sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -24,15 +24,18 @@
  */
 public final class IdentityClientOptions {
     private static final int MAX_RETRY_DEFAULT_LIMIT = 3;
-    private static final String DEFAULT_CACHE_FILE_NAME = "msal.cache";
+    private static final String DEFAULT_PUBLIC_CACHE_FILE_NAME = "msal.cache";
+    private static final String DEFAULT_CONFIDENTIAL_CACHE_FILE_NAME = "msal.confidential.cache";
```

Comment:
Is this cache name going to be same across all languages ?

Reply:
Yes. This is the name for Windows regardless of languages.